packages/integration-tests/src/helper.ts

```diff
@@ -0,0 +1,89 @@
+import { execSync } from "child_process";
+import { ethers } from "ethers";
+import { promises as fs } from "fs";
+import * as path from "path";
+import * as request from "supertest";
+import { Provider } from "zksync-web3";
+
+import { environment, localConfig } from "./config";
+import { Logger } from "./entities";
+
+import type { BaseProvider } from "@ethersproject/providers/src.ts/base-provider";
+
+export class Helper {
+  async txHashLogger(txType: string, txValue: string, tokenName?: string) {
```

Comment:
This is a function, please rename it to `logTransaction` or something.

Reply:
updated